<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmWeighTable</class>
 <widget class="QWidget" name="FrmWeighTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,9">
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QGridLayout" name="gridLayout_11" columnstretch="0,0,0,0,0,0,0,0,0,0">
      <item row="0" column="7">
       <spacer name="horizontalSpacer_18">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>结束日期:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="editField">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>字段内容:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QPushButton" name="btnExport">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>导出Excel</string>
        </property>
        <property name="text">
         <string>导出Excel</string>
        </property>
        <property name="icon">
         <iconset resource="../../main.qrc">
          <normaloff>:/image/excel.png</normaloff>:/image/excel.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>开始日期:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="btnQuery">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>字段名称:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="dateStart">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QPushButton" name="btnRefresh">
        <property name="toolTip">
         <string>刷新</string>
        </property>
        <property name="text">
         <string>刷新</string>
        </property>
        <property name="icon">
         <iconset resource="../../main.qrc">
          <normaloff>:/image/刷新.png</normaloff>:/image/刷新.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="dateEnd">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="cbxField">
        <item>
         <property name="text">
          <string>车牌号码</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="0">
       <item row="0" column="0">
        <widget class="GbsTableWidget" name="tableWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_4"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GbsTableWidget</class>
   <extends>QWidget</extends>
   <header>gbstablewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../main.qrc"/>
 </resources>
 <connections/>
</ui>
